packages/db-model/src/model.ts

```diff
@@ -72,86 +70,96 @@ export function createBaseModel(resolver?: ConnectionResolver) {
     }
 
     /**
-     * Generic query builder.
-     *
-     * @param {(qb: Knex | Transaction) => QueryBuilder} callback
-     * @param {Transaction} [trx]
-     * @returns {Promise<T[]>}
-     */
-    public static async buildQuery<T>(
```

Comment:
This actually gives you an ability to build your query based on Knex when the model provided methods do not fit your needs.
![image](https://user-images.githubusercontent.com/3315763/79762466-8a530c00-8342-11ea-9453-48c5e2427c14.png)